extension/ts/constants.ts

```diff
@@ -1,5 +1,3 @@
-const FORM_RENDER_EVENT = "library-thing-form-rendered";
-const SHOW_TOAST_EVENT = "show-blt-toast";
-const RELEVANT_TAGS = ["textarea", "input", "select"];
+const FORM_DATA_ELEMENT_TAGS = ["textarea", "input", "select"];
```

Comment:
It is my dream that this file will not exist anymore.

I think maybe a `bookForm.ts` type thing? Then we can do away with `util.ts` as well, because every time we try to get all the tags by class name, it's these or a subset. Like `constants` and `util` are coupled right now, so we might as well consolidate.